src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -12764,10 +12764,7 @@ Unit* Creature::SelectVictim()
         return NULL;
 
     if (target && _IsTargetAcceptable(target) && CanCreatureAttack(target))
-    {
-        SetInFront(target);
         return target;
```

Comment:
Maybe set caster in front of target in spell:update like I did in this PR but with other spells ? I just don't know which spells need to track their focustarget :/
Will take a look at this